app/src/main/java/org/fossasia/openevent/general/event/EventUtils.kt

```diff
@@ -10,6 +10,9 @@ import timber.log.Timber
 
 object EventUtils {
 
+    val SIMILAR_EVENTS: String = "similarEvents"
+    val EVENTS: String = "events"
```

Comment:
Where are these needed?

Reply:
These are needed in EventsFragment and SimilarEventsFragment respectively. I have used the same adapter and ViewHolder for both the recycler views.